### ISSUE TITLE
Plain text passwords exported when enablePasswordRetrieval = false

### ISSUE THREAD

**Issue by Navilois** [fordnn/usersexportimport#12]
ForDnn.UsersExportImport export plain text passwords, even though enablePasswordRetrieval="false". This is true for 1.2.1 as well as 1.2.5.

How is this even possible? I thought passwordFormat="Hashed" means it's a one way street?!

Here's my web.config membership section
`<add name="AspNetSqlMembershipProvider" type="System.Web.Security.SqlMembershipProvider" connectionStringName="SiteSqlServer" enablePasswordRetrieval="false" enablePasswordReset="true" requiresQuestionAndAnswer="false" minRequiredPasswordLength="4" minRequiredNonalphanumericCharacters="0" requiresUniqueEmail="true" passwordFormat="Hashed" applicationName="DotNetNuke" description="Stores and retrieves membership data from the local Microsoft SQL Server database" />`

**Comment by Navilois** [comment]
This is only true for very old users, back when passwordFormat was not hashed, but encrypted I believe. Also DNN version around 7.0.4 or even older. All newer users have the password field empty in the csv file.

**Comment by fordnn** [comment]
Dear Alois,

Thank you for this info. It's really interesting. "Hashed" means passwords are encrypted one-way. What version of DNN this happened? Are you sure it was not just changed in web.config? Please run this query to confirm this:

SELECT DISTINCT PasswordFormat 
FROM aspnet_Membership

Based on results you can see:

0=clear, 1=Hashed, 2=Encrypted

Best regards,
Serg

**Comment by Navilois** [comment]
PasswordFormat is 1 for all users. I'm running 9.8.1.

I checked both at dev and production environment to make sure I wasn't wrong with the passwordRetrieval setting. Also searched web.config for "password", only occurs in the membership provider section of web.config. I scoured through related DNN tables to find old entries, maybe even plain text passwords - nothing so far.

I assumed this one is the related setting to passwordRetrieval="false" in web.config:
https://github.com/fordnn/usersexportimport/blob/705cff39d2f8b146aba1b3e0aad8776bb723212b/Components/ExportController.cs#L178
Maybe there's an old setting in the DB overriding this? This website has been upgraded numerous times from ancient DNN versions.
